src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
If you still want that to be strongly typed, I will have to add contracts for the mappings I need, and everyone else will pretty much have to come in and add theirs, since this is not definied in the API, but in the textual mapping values.